tools/stanford/flowsheet_cleaner.py

```diff
@@ -85,13 +103,22 @@ def get_new_sources(root: str, child: str) -> Set[str]:
                     continue
                 if new_row["observation_concept_id"] == "0":
                     new_concepts.add(new_row["observation_source_value"])
+                    new_concepts.add(new_row["observation_parent_value"])
+
+                    new_relationships.add(
+                        (
+                            new_row["observation_source_value"],
+                            new_row["observation_parent_value"],
+                        )
+                    )
+
                 if new_row["unit_concept_id"] == "0":
                     new_concepts.add(new_row["unit_source_value"])
     except Exception as e:
         traceback.print_exc()
         raise RuntimeError("Failed " + root + " , " + child, e)
 
-    return new_concepts
+    return new_concepts, new_relationships
 
 
 def correct_rows(
```

Comment:
In documentation, anytime a claim is made, we should minimally document why that claim is true.  For this function, for example, why are some mappings incorrect? What is a description of the fix in plain language for? etc